pkgs/_cfe_macros/pubspec.yaml

```diff
@@ -0,0 +1,92 @@
+name: _cfe_macros
+publish-to: none
+description: Macro support for the CFE.
+resolution: workspace
+
+environment:
+  sdk: ^3.6.0-48.0.dev
+
+dev_dependencies:
+  dart_flutter_team_lints: ^3.0.0
+  front_end: any
+  frontend_server: any
+  macros: ^0.1.2-main.4
+  test: ^1.25.0
+
+# Update by running: tool/set_sdk_version <git ref>
+dependency_overrides:
```

Comment:
I believe all these overrides can/should exist only in the root (workspace) pubspec.yaml